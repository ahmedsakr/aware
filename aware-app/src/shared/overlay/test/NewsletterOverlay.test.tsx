import React from 'react';
import { shallow } from 'enzyme';

import NewsletterOverlay from './NewsletterOverlay';

describe('NewsletterOverlay', () => {
    it('should render correctly', () => {

        let content = () => {
            return (
                <div>
                    testing body content!
                </div>
            );
        }

        let footer = () => {
            return (
                <div>
                    testing footer content!
                </div>
            );
        }

        const component = shallow(
            <NewsletterOverlay
                name="Aware Newsletter"
                title="Subsribe to our Newsletter"
                content={content()}
                footer={footer()} />
        );

        expect(component).toMatchSnapshot();
    });
});