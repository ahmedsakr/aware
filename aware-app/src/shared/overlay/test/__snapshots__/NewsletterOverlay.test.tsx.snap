version https://git-lfs.github.com/spec/v1
oid sha256:8fa392be8321e4845b04b7a678b3408afce90ea5bc277a96a57caafff545e014
size 1605
